ml-agents/mlagents/trainers/models.py

```diff
@@ -4,7 +4,7 @@
 import tensorflow as tf
 import tensorflow.contrib.layers as c_layers
 
-logger = logging.getLogger("mlagents.envs")
+logger = logging.getLogger("mlagents.trainers")
```

Comment:
How come this is not `mlagents_trainers` ?

Reply:
This perhaps should be part of a bigger discussion. In the current version there is no `mlagents_trainers`, there is only `mlagents_envs`, and `mlagents.trainers`.

Reply:
It just does not make sense to me to keep `mlagents.trainers` if there is no longer a `mlagents` package...